test/unit/gateways/card_stream_test.rb

```diff
@@ -27,6 +27,18 @@ def setup
       description: 'AM test purchase'
     }
 
+    @visacredit_three_ds_options = {
+      threeds_required: true,
+      three_ds_version: '2.1.0',
+      three_d_secure: {
+        enrolled: 'true',
+        authentication_response_status: 'Y',
+        eci: 0o5,
+        cavv: 'Y2FyZGluYWxjb21tZXJjZWF1dGg',
+        xid: '00000000000004717472'
```

Comment:
Let's make this an example 3ds2 style uuid-formatted value to prevent versioning confusion.